packages/reassure-cli/src/utils/printer.ts

```diff
@@ -0,0 +1,69 @@
+import chalk from 'chalk';
+import { CHALK } from '../constants';
+
+const prefix = chalk.hex(CHALK.colors.primary)('reassure-cli: ');
+
+const hello = `
+========================================================================
+======================== Welcome to reassure-cli =======================
+========================================================================
+
+          .(#################          (################(        
+          *##################(       (######################      
+          *#########//////(##      .########/       (########,    
+          *#######                *#######            (#######    
+          *#######                #######/             #######/   
+          *#######               (############################(   
+          *#######               (#############################   
+          *#######                #######/                        
+          *#######                ########,                       
+          *#######                 #########                      
+          *#######                  (################(######      
+          *#######                    /#####################(     
+          *#######                        (################(      
+                                                       
+`;
+
+const bye = `
+========================================================================
+====================== built with ❤️  at callstack ======================
+================================ --- ===================================
+==================== find us @ https://callstack.io ====================
+============= or on GitHub @ https://github.com/callstack ==============
+========================================================================
+`;
+
+export const printHello = (logLevel: string | undefined) => {
+  if (logLevel === 'silent' || logLevel === 'no-greetings') return;
+  return console.log(chalk.hex(CHALK.colors.primary)(hello));
+};
+
+export const printCiSetupHint = (logLevel: string | undefined) => {
+  if (logLevel === 'silent') return;
+
+  console.log(chalk.hex(CHALK.colors.dim)('Finished initalizing new reassure testing environment.'));
+  console.log(chalk.hex(CHALK.colors.dim)('Please refer to our CI guide in order to set up your pipelines.'));
+  console.log(
+    chalk.hex(CHALK.colors.dim)('Find more @ https://callstack.github.io/reassure/docs/installation#ci-setup')
+  );
+};
+
+export const printBye = (logLevel: string | undefined) => {
+  if (logLevel === 'silent' || logLevel === 'no-greetings') return;
+  return console.log(chalk.hex(CHALK.colors.primary)(bye));
+};
+
+export const printLog = (logLevel: string | undefined, ...args: any[]) => {
+  if (logLevel === 'silent' || logLevel === 'default') return;
+  return console.log(prefix, chalk.hex(CHALK.colors.dim)(args));
```

Comment:
Why log is priting in dim colors?

Reply:
because log pertains to the least important procedural data. It's just looked better in logs to be perfectly honest. 

The idea is simple: 
- logs are only if you need more details 
- warns are messages which may be helpful and are visible by default
- errors are very important for the user therefore can only be removed by running `--silent`